src/Raven.Client/Documents/LinqExtensions.cs

```diff
@@ -861,6 +861,20 @@ public static IOrderedQueryable<T> OrderByScore<T>(this IQueryable<T> self)
             return (IOrderedQueryable<T>)queryable;
         }
 
+        /// <summary>
+        /// Perform an initial sort by lucene score.
+        /// </summary>
+        public static IOrderedQueryable<T> ThenByScore<T>(this IQueryable<T> self)
```

Comment:
We need to change this to `IOrderedQueryable` IIRC

Reply:
I'm referring to `this IQueryable<T> self`

Reply:
fixed